testgrid/config/config.yaml

```diff
@@ -4137,6 +4138,8 @@ dashboards:
     test_group_name: maintenance-ci-testgrid-config-upload
     code_search_url_template:
       url: https://github.com/kubernetes/test-infra/compare/<start-custom-0>...<end-custom-0>
+    alert_options:
+      alert_mail_to_addresses: 'gke-kubernetes-engprod+alerts@google.com'
```

Comment:
not sure which address we want to use, but *somebody* should get a ping when the config updater fails

Reply:
feel free to add my personal email here :-)